lib/src/flutter_sms_platform.dart

```diff
@@ -32,9 +32,12 @@ class FlutterSmsPlatform extends PlatformInterface {
     _instance = instance;
   }
 
+  ///
+  ///
   Future<String> sendSMS({
     required String message,
     required List<String> recipients,
+    required bool sendDirect,
```

Comment:
This should probably be optional and default to false so it's not a breaking change.